c++/src/kj/compat/http.c++

```diff
@@ -4077,6 +4556,28 @@ public:
     return getClient(parsed).openWebSocket(path, headersCopy);
   }
 
+  kj::Promise<ConnectResponse> connect(kj::StringPtr host, const HttpHeaders& headers) override {
+    auto hostCopy = kj::str(host);
+    auto headersCopy = headers.clone();
+
+    // TODO(now): Verify that this is correct.
+    auto hostHeader = KJ_REQUIRE_NONNULL(headersCopy.get(HttpHeaderId::HOST),
+                                         "Host header is required for connect");
+    auto parsed = Url(kj::str("http"),
```

Comment:
Maybe easier to do:

```
Url parsed;
parsed.scheme = kj::str("http");
parsed.host = kj::str(hostHeader)`
```

Reply:
Just removing this implementation as suggested.